README.md

```diff
@@ -159,7 +159,7 @@ acquire this dependency.
 These instructions assume you're targeting `x64-windows`; you can change this constant below to target other
 architectures.
 
-1. Install [CMake][] 3.16.5 or later, [Ninja][] 1.10.0 or later, and Visual Studio 2019 16.7 Preview 3 or later.
+1. Install [CMake][] 3.16.5 or later, [Ninja][] 1.10.0 or later, and Visual Studio 2019 16.8 Preview 1 or later.

```

Comment:
I note that we're now testing with the bundled versions of CMake/Ninja/LLVM. Should we change this to something like "Install VS including CMake and LLVM (you may use newer versions of CMake, Ninja, and/or LLVM but expect to do some more manual configuration)" to more precisely reflect the new requirement?

Reply:
I've adjusted the wording here, although differently from your suggestion.